lua/telescope/actions/mt.lua

```diff
@@ -45,18 +45,35 @@ action_mt.create = function(mod)
     end,
 
     __add = function(lhs, rhs)
-      local new_actions = {}
+      local new_action = setmetatable({}, action_mt.create())
       for _, v in ipairs(lhs) do
-        table.insert(new_actions, v)
+        table.insert(new_action, v)
+        new_action._func[v] = lhs._func[v]
+        new_action._static_pre[v] = lhs._static_pre[v]
+        new_action._pre[v] = lhs._pre[v]
+        new_action._replacements[v] = lhs._replacements[v]
+        new_action._static_post[v] = lhs._static_post[v]
+        new_action._post[v] = lhs._post[v]
```

Comment:
can we write little helper function to copy these over? we do the same thing twice.

Also, have we documented what all the fields do? I don't remember if we did that or not. Could be helpful.